src/Core/src/Response.php

```diff
@@ -80,7 +88,17 @@ public function resolve(?float $timeout = null): bool
                 if ($chunk->isTimeout()) {
                     return false;
                 }
-                if ($chunk->isFirst()) {
+                if ($fullResponse) {
+                    if ($chunk->isLast()) {
+                        $this->streamable = false;
+
+                        break;
+                    }
+
+                    if ($this->streamable && '' !== $chunk->getContent()) {
```

Comment:
Can you read a chunk multiple times?

Reply:
yes, you can read multiple time the same chunk, but can't rewind the stream